### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by williamlixu**
**Summary**
Just a small cleanup to the PR template.

**Testing**
Used GitHub markdown preview and looks correctly formatted to me.

**Checklist**

- [x] Code builds successfully
- [x] All tests pass

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Closes #(issue)
 
 **Description**
 
-This PR introduces... (write a simple description of the added feature or solution to the bug)
+Wwrite a simple description of the added feature or solution to the bug.
```
Comment (margeobur) [inline .github/pull_request_template.md]: You got two w's my dawg

**Review by margeobur — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (margeobur) [reply .github/pull_request_template.md]:
```suggestion
Write a simple description of the added feature or solution to the bug.
```

**Review by margeobur — APPROVED** [review]
LGTM

**Review by jwar663 — APPROVED** [review]
LGTM

**Review by Curtiss02 — APPROVED** [review]
👍

**Pull request merged**